rosidl_runtime_c/include/rosidl_runtime_c/u16string_functions.h

```diff
@@ -67,6 +67,24 @@ rosidl_runtime_c__U16String__are_equal(
   const rosidl_runtime_c__U16String * lhs,
   const rosidl_runtime_c__U16String * rhs);
 
+/// Copy rosidl_runtime_c__U16String structure content.
+/**
+ * This functions performs a deep copy, as opposed to the shallow copy that
+ * plain assignment yields.
+ *
+ * \param[out] input a pointer to a rosidl_runtime_c__U16String structure
```

Comment:
```suggestion
 * \param[in] input a pointer to a rosidl_runtime_c__U16String structure
```

Reply:
Fixed in https://github.com/ros2/rosidl/pull/650/commits/ab79f66aac8c5e876481596bb9cb06eb6a1e6e3b.